jMolecules/associations_vo.md

```diff
@@ -1,4 +1,11 @@
 # Entities in Value Objects werden nicht richt gemapped
+
+## Meta
+
+Status: In Frage
+Fragen: Es ergibt aus DDD-Sicht keinen Sinn, aus ValueObjects Entitäten zu referenzieren, da erstere immutable sind und letztere nicht. Ich vermute die Verirrung kommt daher, dass JPA es erlaubt aus einem Embeddable Entitäten zu referenzieren. Ich würde empfehlen, aus dem ValueObject lediglich eine ID zu referenzieren, üblicherweise auch jediglich eine eines Aggregates, da Entitäten in DDD eigentlich nur lokale Identität haben.
```

Comment:
Evans schreibt in seinem Buch im Kapitel zu Value Objects, dass diese Entities referenzieren können. Ich weiß gerade nicht, was Vernon dazu sagt. Ich habe gerade aber keins der Bücher vor mir liegen. Es ist auch gut möglich, dass ich Evans in diesem Fall missverstanden habe. Mit der Modellierung müsste ich mich nämlich nochmal genauer auseinandersetzen, da ich nach dem Arbeiten mit diesen Value Objects festgestellt habe, dass meine Modellierung verbesserungswürdig ist.

Wenn ich Evans richtig verstanden habe, wäre für mich die Frage, ob jMolecules DDD, wie im blauen Buch, umsetzen soll oder nicht.